web/src/peer_data.ts

```diff
@@ -82,8 +82,8 @@ export function get_subscribers(stream_id: number): number[] {
     return [...subscribers.keys()];
 }
 
-export function set_subscribers(stream_id: number, user_ids: number[]): void {
-    const subscribers = new LazySet(user_ids);
+export function set_subscribers(stream_id: number, user_ids?: number[]): void {
+    const subscribers = new LazySet(user_ids ?? []);
```

Comment:
I don’t think we’re expecting callers to pass `undefined` here; if they do, we should fix the callers.